src/components/PopupRevisionOverview/PopupRevisionOverview.js

```diff
@@ -885,11 +911,12 @@ const Belangen = ({ label, object, type, containsChanges, placeholder }) => {
 }
 
 /**
+ * Component that renders the RelationsConnectionsText component which displays two containers containing the connected relations between objects.
  *
- * @param {object} props
- * @param {object} props.originalObject - Contains the original object
- * @param {object} props.objectChanges - Contains the object that has changes
- * @returns A section that indicates the changes in connections
+ * @component
+ *
+ * @param {object} originalObject - Contains the original object
+ * @param {object} objectChanges - Contains changes saved within the object.
```

Comment:
I would revert this back to the original documentation line. 'changes within the object' feels a bit confusing

Reply:
I've reverted the changes.